charts/fluent-operator/templates/fluent-operator-servicemonitor.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.operator.enable }}
```

Comment:
copied the [fluentbit-servicemonitor.yaml](https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/templates/fluentbit-servicemonitor.yaml) and adjusted the labels